### PR TITLE
add account fields to notification

### PR DESCRIPTION
**Pull request by mikiquantum**
Issue Link: #489

### CODE REVIEW

File: notification/notification_test.go [hunk 1]
```diff
@@ -50,7 +50,8 @@ func (m mockConfig) GetReceiveEventNotificationEndpoint() string {
 
 func TestWebhookSender_Send(t *testing.T) {
 	docID := utils.RandomSlice(32)
-	cid := utils.RandomSlice(32)
+	accountID := utils.RandomSlice(6)
```
Comment (lucasvo) [inline notification/notification_test.go]: Why is the accountid so short?

Comment (vimukthi-git) [reply notification/notification_test.go]: its a centID

Comment (mikiquantum) [reply notification/notification_test.go]: the account_id is the centID and as of today it is 6 bytes long.

Comment (vimukthi-git) [reply notification/notification_test.go]: btw @mikiquantum could you use the `identity.CentIDLength` constant for this or use `identity.RandomCentID` function? better for readability and understanding.

**Pull request merged**